features/beepBoop.js

```diff
@@ -1,6 +1,6 @@
 // const { directMention } = require("@slack/bolt");
 // const { directMessage, anyOf } = require("../middleware");
-const boopNo = require("../lib/beepBoop_lib");
+const boopNo = require("../services/beepBoopServ");
```

Comment:
This change doesn't pertain to the connection pools, but was an error that got to production.